Source/Core/Core/PowerPC/TableGen.cpp

```diff
@@ -0,0 +1,459 @@
+#include <bitset>
+#include <cstdlib>
+#include <deque>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include <variant>
+#include <vector>
+
```

Comment:
Everything in here should be namespaced under the PowerPC namespace

Reply:
All of this is supposed to be compiled before the Dolphin code, as it is generating code for that build step (which is why it doesn't include any Dolphin headers, even if that were useful), so I'm not sure that makes sense. I'm not sure this is the right place to but this code though.